src/web/src/pages/CoursePage.vue

```diff
@@ -25,18 +25,25 @@
     </b-row>
     <b-button :to="backRoute">Back</b-button>
   </div>
-  <div v-else>
-    <b-spinner></b-spinner>
-    <strong class="m-2">Loading Course...</strong>
-  </div>
+  <CenterSpinner 
+    v-else
+    :height = "80" 
+    :fontSize = "1.4"
+    :loadingMessage = "Courses"
```

Comment:
This colon needs to be removed, its interpreting `"Courses"` as js code so the message isn't being displayed.

Reply:
You're good I was being ridiculously careless. The spinner for subject explorer should also be fixed. Thanks for pointing these out!